cli/lib/kontena/cli/nodes/labels/add_command.rb

```diff
@@ -2,17 +2,17 @@ module Kontena::Cli::Nodes::Labels
   class AddCommand < Kontena::Command
     include Kontena::Cli::Common
 
-    parameter "NODE_ID", "Node id"
+    parameter "NODE", "Node"
```

Comment:
You mean in `kontena node list`? Perhaps, currently it will just show `(initializing)` in this case, which makes it very hard to get rid of that node if it never manages to update.

This PR or separate issue?